s3blobs/src/play/modules/s3blobs/S3Blobs.java

```diff
@@ -19,22 +19,8 @@ public void onApplicationStart() {
 		}		
 
 		Logger.info("Starting the S3Blobs module");
-		
-		if (!Play.configuration.containsKey("aws.access.key")) {
-			throw new ConfigurationException("Bad configuration for s3: no access key");
-		} else if (!Play.configuration.containsKey("aws.secret.key")) {
-			throw new ConfigurationException("Bad configuration for s3: no secret key");
-		} else if (!Play.configuration.containsKey("s3.bucket")) {
-			throw new ConfigurationException("Bad configuration for s3: no s3 bucket");
-		}
 		S3Blob.s3Bucket = Play.configuration.getProperty("s3.bucket");
-		S3Blob.serverSideEncryption = ConfigHelper.getBoolean("s3.useServerSideEncryption", false);
-		String accessKey = Play.configuration.getProperty("aws.access.key");
-		String secretKey = Play.configuration.getProperty("aws.secret.key");
-		AWSCredentials awsCredentials = new BasicAWSCredentials(accessKey, secretKey);
-		S3Blob.s3Client = new AmazonS3Client(awsCredentials);
-		if (!S3Blob.s3Client.doesBucketExist(S3Blob.s3Bucket)) {
-			S3Blob.s3Client.createBucket(S3Blob.s3Bucket);
-		}
+		S3Blob.serverSideEncryption = ConfigHelper.getBoolean("s3.useServerSideEncryption", true);
+		S3Blob.s3Client = new AmazonS3Client();
```

Comment:
Should we keep this backwards compatible for now? 

E.g check for the credentials, but if not present then use IAM roles? 

Also I'm not sure how this will behave when testing locally either? There was also a ticket to try and add better test support for S3 so would be good to resolve that too if we can? Or can add this as a 2nd ticket.

Reply:
Having different credentials for different AWS functionalities will not improve the developer's experience. if we rely on the credential chain, the developer can pass its own credentials through env variables. and all interactions with AWS APIs will use the same identity.

Reply:
In terms of testing, I think it might be better to create/search for solutions that mock out AWS calls.

Reply:
In terms of having an orderly rollout. testing and releasing a new version requires a version bump in qudinapp that needs to be with the change in `application.conf`. since the IAM policy already is in place. rolling this out will be seamless, (we need to copy the all objects across before releasing this)